extensions/test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM registry.ci.openshift.org/rhcos-devel/rhel-coreos:latest
+#We want to test it's only using the extensions repo
+RUN rm -rf /etc/yum.repos.d/*
+ADD ext.repo /etc/yum.repos.d/extention.repo
+#Install usbguard as provided by the repo
+RUN rpm-ostree install usbguard
+RUN usbguard
```

Comment:
Maybe better as `usbguard --help`?  Oh weird...just invoking `usbguard` prints the usage but also exits successfully, but `usbguard --help` exits with code 1.  And annoyingly it doesn't have a no-op success command like `usbguard --version`.

Oh well, OK as is I guess!